llvm/test/MC/X86/sm4-evex-32-att.s

```diff
@@ -0,0 +1,224 @@
+// RUN: llvm-mc -triple x86_64-unknown-unknown --show-encoding %s | FileCheck %s
```

Comment:
The same to below.

Reply:
[23d4bfc](https://github.com/llvm/llvm-project/pull/113402/commits/23d4bfcabdca8968cba600e2fc0b440121990ef5)